README.md

```diff
@@ -1,6 +1,6 @@
 # Stack Learner - Web Design Master Class
 
-In this repository, you will find source codes, explanations, and other resources for **Web Design Master Class**. It is an open-source repository and feel free to add documentation, examples, and other references when necessary. Please create issues when you get confused or if something does not work. Other members will help you to solve your issues.
+In this repository, you will find source codes, explanations, and other resources for **Web Design Master Class**. It is an open-source repository and feels free to add documentation, examples, and other references when necessary. Please create issues when you get confused or if something does not work. Other members will help you to solve your issues.
```

Comment:
#

Reply:
Your requested change is unnecessary considering my requested changes. A repository can't feel free to add anything; A human does.